post-processing/post_processing_config.yaml

```diff
@@ -49,8 +49,6 @@ column_types:
   cpus_per_task: "int"
 
 # Specify which columns to export to csv file
-csv_export:
-  [tasks,
-  flops_value,
-  Triad_unit,
+additional_columns_to_csv:
+  [flops_value,

```

Comment:
This is not an additional column, it's the `y_axis`, therefore it will be printed out anyway. I think you're mixing two different configs. 
This config file is just an example, so you can either add an imaginary column name here, that is not mentioned anywhere else in the config, or make it vaguely correct for the Sombrero perflog, currently sitting in the `tutorial` folder, and pick a random column from there - again, it shouldn't be mentioned anywhere else in the config.